### ISSUE TITLE
Hoping for pity

### ISSUE THREAD

**Issue by maekillam** [tombowditch/telly#68]
I really want to use this program as I think it is exactly what I need. I am fumbling blindly , however, because I know little about how it works and enough about programming to be dangerous. I have two choices to install. A computer running windows 10 home (where I have my PLEX already) or a QNAP TS-469 Pro (where I could easily  install an additional PLEX server) . I tried the telly-windows-amd64 executable in windows and a black screen pops up for a second with nothing else occurring. I see reference to command line for the playlist but cannot figure out how or where to do this. I tried installing a few of the versions on the QNAP which made sense to me by sh through putty but keep getting the error that it can't execute a binary. I understand this may be way out of my league but I am hoping against hope someone might take me through either system step by step. Nothing ventured nothing gained I guess. Thank you....

**Comment by HazCod** [comment]
Try opening commandprompt, going to the directory where telly resides and executing it with the correct arguments. Just clicking it doesn't supply enough arguments, making it to exit.

Perhaps someone on Windows can chime in and give you the commands.

**Comment by haxcop** [comment]
Hi @maekillam is easy 
for example:
if you have the telly program into your download folder it will look something like this
>c:\users\YOU\downloads\telly-windows-amd64.exe
Now you can run it in this way: Open PowerShell as Administrator and follow the below line
`c:\users\YOU\downloads\telly-windows-amd64.exe  -playlist="YOUR M3U URL OR FILE LOCATION HERE WITH THE QUOTES"`
this will do the job but probably will load more than 400+ channels and Plex will hang trying to get them all, so here you can make use of the other `telly` Arguments like `-useregex="'Ireland'"` to filter only the Irish TV Group channels: you can add other groups like `-useregex="'group A', 'group B','etc'"
The general documentation of this is at the Readme of the repro - is really easy when you try and get familiar with it. 
if you need more help please join us into the discord channel also mentioned at the bottom of the readme .

**Comment by shaneosullivanky** [comment]
Or if Powershell is a bit advanced, you can simply use CMD prompt (DOS command).

Depending on which version of Windows you have, just search for CMD and then follow @haxcop ’s instructions.

Let us know how you get on and we can walk you through it step by step.

**Comment by maekillam** [comment]
Thank you for the help and responses. I followed the instructions. I have an epg service which can whittle down an m3u file and give a new URL for it which is what I used. Appaantly all I was missing was the quotes when I tried it before. When I ran it this time I got the message: 

C:\WINDOWS\system32>c:\telly-windows-386.exe -playlist="http://iptv-epg.com/some.m3u"
[telly] [info] booting telly v0.4.5
[telly] [error] Could not create file: C:\Users\THEKIL~1\AppData\Local\Temp ; open C:\Users\THEKIL~1\AppData\Local\Temp: is a directory

I tried with both 386 and AMD64 files. I saw something similar in the issue threads. I am assuming this is still a problem? Or is there a workaround or an updated build I can try? 

If windows isn't working is there anyone who can help with the QNAP SSH?
